devolutions-gateway/src/api/network_scan/mod.rs

```diff
@@ -0,0 +1,12 @@
+use crate::DgwState;
+use axum::Router;
+
+pub mod ipconfig;
+pub mod scan;
+
+pub fn make_router<S>(state: DgwState) -> Router<S> {
+    Router::new()
+        .route("/scan", axum::routing::get(scan::handler))
+        .route("/ipconfig", axum::routing::get(ipconfig::handler))
```

Comment:
issue: The endpoint should be `/jet/net/config`
```suggestion
        .route("/config", axum::routing::get(ipconfig::handler))
```